hedera-node/hedera-app/src/main/java/com/hedera/node/app/HederaApp.java

```diff
@@ -85,7 +86,8 @@
             ExpiryModule.class,
             ServiceModule.class,
             QueryWorkflowModule.class,
-            HandleWorkflowModule.class
+            HandleWorkflowModule.class,
+            IngestModule.class
```

Comment:
This is needed because of some other dependencies?

Reply:
Yep, needed to bind `SignaturePreparer`.

Reply:
Removed this, ok to install this module only via `IngestComponent`.